pkg/plugins/layout.go

```diff
@@ -205,6 +206,16 @@ func (dev Disk) String() string {
 	return dev.Device
 }
 
+// IsGPT returns true if the disk is GPT partitioned
+func (dev *Disk) IsGPT(console Console) (bool, error) {
+	out, err := console.Run(fmt.Sprintf("fdisk -l %s", dev.Device))
```

Comment:
Im wondering if fdisk is available during initramfs or this is a new dep... cant we use sgdisk/sfdisk for the same?

Reply:
umm seems the fdisk is not explicitely added to the initramfs, which doesnt mean that its not there, but its a possibility: https://github.com/kairos-io/immucore/blob/master/dracut/28immucore/module-setup.sh#L28

Reply:
seems like the same can be done by checking:

`console.Run(fmt.Sprintf("blkid %s -s PTTYPE -o value", dev.Device))`

Which would return either `gpt` or `dos

Reply:
thanks, will switch to blkid then

Reply:
done